admin-dev/themes/new-theme/template/light_display_layout.tpl

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="{$iso}">
+<head>
+  {$header}
+</head>
+
+<body
+  class="lang-{$iso_user}{if $lang_is_rtl} lang-rtl{/if} {$smarty.get.controller|escape|strtolower}{if $collapse_menu} page-sidebar-closed{/if}"
+  {if isset($js_router_metadata.base_url)}data-base-url="{$js_router_metadata.base_url}"{/if}
+  {if isset($js_router_metadata.token)}data-token="{$js_router_metadata.token}"{/if}
+>
+
+<div id="main-div" class="light_display_layout">
```

Comment:
it feels strange to add a new layout just because of a scrollbar issue... but then maybe it will have other uses cases in the future ?

Reply:
The idea is also to create a dedicated clean layout for those use cases, which is cleaner than a common single layout with `if` blocks everywhere

And I couldn't modify the current generic `layout.tpl` without risking introducing some regression

Reply:
With twig we probably have used a different layout (maybe one that extends the original one) So it makes sense to me to have two different layouts for two different ways of displaying the BO

Reply:
sounds good 👍